src/ray/core_worker/common.h

```diff
@@ -127,7 +127,8 @@ struct ActorCreationOptions {
   const std::vector<std::string> dynamic_worker_options;
   /// Whether to keep the actor persistent after driver exit. If true, this will set
   /// the worker to not be destroyed after the driver shutdown.
-  const bool is_detached = false;
+  /// nullptr is not specified.
+  std::shared_ptr<bool> is_detached = nullptr;
```

Comment:
Not sure if it's a best practice to use a pointer for such enums.

Reply:
You're right. The reason for choosing shared_ptr is that it's easy to use in cython. :)